### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by piesieomane**
This pull request consists of GitHub workflows and linters to help document my project.
The homepage talks about myself and my career journey

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<body>
+    <nav class="navbar">
+        <div>
+            <h3 class="my-logo">My logo</h3>
+        </div>
+
+       <div class="nav-right">
+           <img src="images/Enabled.png" alt="normal button">
+       </div>
+    </nav>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly wrap this section with a header tag, only the hamburger menu on the right should have a nav tag.
- [ ] Kindly ensure your logo has a tag, it should be clickable.
- [ ] Kindly add the background images to your design to match the design given

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image|  image

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @piesieomane , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted Your project using the wrong submission link. You submitted as Portfolio Application - Project 2, instead of the first project. please check that and use the right submission link. 😄 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+           <a href="https://www.linkedin.com/feed/" class="github-icon icon-right">
+            <img src="images/github.svg" alt="">
+           </a> 
+           <a href="https://www.linkedin.com/feed/" class="linkdin-icon icon-right">
+            <img src="images/Linkedin-icon.svg" alt="">
+           </a> 
+           <a href="https://www.linkedin.com/feed/" class="microsoft-icon icon-right">
+            <img src="images/microsoft.svg" alt="">
+           </a> 
+        </div>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly use `<ul>` and `<li>` in this section, it's a project requirement.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        </h2>
+        <p class="about">
+            I'm a software developer! I can help you build a product , 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't 
+            hestiate to contact me.
+        </p>
+
+        <div class="social-media">  
+            <p>LET'S CONNECT</p>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly make sure to use an `<a>` tag or a `<button>` tag for this section, since it's a link it needs to be clickable.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @piesieomane , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 3 above)
Comment (zieeco) [reply index.html]: @danmainah that should be <h1-h6> tag,  the `a` or `button` tag is not necessary and not required..

Comment (piesieomane) [reply index.html]: please i have done that

Comment (piesieomane) [reply index.html]: The first reviewer said i should place them in an Anchor tag or a button

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @piesieomane 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**